src/libs/cluster-utils.js

```diff
@@ -82,6 +82,21 @@ export const trimClustersOldestFirst = _.flow(
 
 export const currentCluster = _.flow(trimClustersOldestFirst, _.last)
 
+export const trimAppsOldestFirst = _.flow(
+  _.remove({ status: 'Deleting' }),
+  _.sortBy('createdDate')
+)
+
+export const currentApp = _.flow(trimAppsOldestFirst, _.last)
+
+export const appIsProvisioning = app => {
+  return app && app.status === 'PROVISIONING'
+}
+
+export const appIsDeleting = app => {
+  return app && app.status === 'DELETING'
```

Comment:
Is the status `'DELETING'` or `'Deleting'`, as above?

Reply:
'DELETING' is what swagger says. I'm not finding where 'deleting' occurs in the code though, which line did you see that?